test/syscalls/linux/socket_netlink_util.cc

```diff
@@ -108,5 +108,42 @@ PosixError NetlinkRequestResponse(
   return NoError();
 }
 
+PosixError NetlinkRequestResponseOneshot(
```

Comment:
nit: This is ok, but I'd maybe rename them

NetlinkRequestResponse -> NetlinkRequestResponseMulti
NetlinkRequestResponseOneshot -> NetlinkRequestResponseSingle

Reply:
Ack.